cuqi/utilities/_get_python_variable_name.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from itertools import count
+import re
+
+import warnings
+import cuqi
+
+
+def get_python_variable_name(var):
```

Comment:
I suggest to make this hidden method '_get_python_variable_name', since we do not expect users to use it.